demo/backstop-common.js

```diff
@@ -269,168 +266,156 @@ module.exports = function backstopCommon(baseUrl) {
       },
       {
         label: `Forms/Checkbox group (hint)`,
-        url: storyUrlFor(`forms-checkbox-group--with-hint`),
+        url: componentUrlFor(`checkbox_group/hint`),
         viewports: [{ label: 'form-field', width: 800, height: 400 }],
       },
       {
         label: `Forms/Checkbox group (optional)`,
-        url: storyUrlFor(`forms-checkbox-group--optional-field`),
+        url: componentUrlFor(`checkbox_group/optional`),
         viewports: [{ label: 'form-field', width: 800, height: 400 }],
       },
       {
         label: `Forms/Checkbox group (error)`,
-        url: storyUrlFor(`forms-checkbox-group--with-error`),
+        url: componentUrlFor(`checkbox_group/error`),
         viewports: [{ label: 'form-field', width: 800, height: 400 }],
       },
       {
         label: `Forms/Checkbox group (long)`,
-        url: storyUrlFor(`forms-checkbox-group--with-long-options`),
+        url: componentUrlFor(`checkbox_group/long`),
         viewports: [{ label: 'form-field', width: 800, height: 400 }],
       },
       {
         label: `Forms/Checkbox (basic)`,
-        url: storyUrlFor(`forms-checkbox--basic`),
+        url: componentUrlFor(`checkbox_single/basic`),
         viewports: [{ label: 'form-field', width: 800, height: 400 }],
       },
       {
         label: `Forms/Checkbox (error)`,
-        url: storyUrlFor(`forms-checkbox--with-error`),
+        url: componentUrlFor(`checkbox_single/error`),
         viewports: [{ label: 'form-field', width: 800, height: 400 }],
       },
-      ...buildSamplePageScenarios(
-        'Sample Page/Content page',
-        'sample-pages-examples--content-page'
-      ),
-      ...buildSamplePageScenarios(
-        'Sample Page/Form page',
-        'sample-pages-examples--form-page'
-      ),
-      {
-        label: `Foundations/Colours`,
-        url: storyUrlFor('foundations-colours--page'),
-        viewports: [commonViewports.large],
-        selectors: ['document'],
-      },
+      ...buildSamplePageScenarios('Sample Page/Content page', 'content-sample'),
+      ...buildSamplePageScenarios('Sample Page/Form page', 'form-sample'),
     ],
     dockerCommandTemplate:
       'docker run --rm -i --mount type=bind,source="{cwd}",target=/src backstopjs/backstopjs:{version} {backstopCommand} {args}',
     paths: {
-      engine_scripts: 'engine_scripts',
-      bitmaps_reference: 'backstop_data/bitmaps_reference',
-      bitmaps_test: 'backstop_data/bitmaps_test',
-      html_report: 'backstop_data/html_report',
-      ci_report: 'backstop_data/ci_report',
+      engine_scripts: 'visual-regression/engine_scripts',
+      bitmaps_reference: 'visual-regression/backstop_data/bitmaps_reference',
+      bitmaps_test: 'visual-regression/backstop_data/bitmaps_test',
+      html_report: 'visual-regression/backstop_data/html_report',
+      ci_report: 'visual-regression/backstop_data/ci_report',
     },
-    report: ['browser'],
+    report: ['html'],
```

Comment:
I liked it backstop would open the browser right after the tests were complete. I do realise that's more of a personal preference thing, so I'm happy to run `npm run backstop:reference` instead

Reply:
Oh yeah! I think I needed to tweak this to run against the demo app, but let me see if I can't maintain that behaviour locally.